Controller/direct message

Mavo39 opened the pull request:
[Controller/DirectMessage]

Edit layouts on Add item, Edit item and Donated-items page
Add to check uploaded photo with using jQuery

Mavo39 committed:
[Controller/DirectMessage] Edit layouts and Add jQuery
Changes:

Add script tags to use jQuery
Edit layouts on Donated-item page, add item, and edit item page

Mavo39 committed:
Merge branch 'main' into Controller/DirectMessage

Mavo39 committed:
Merge branch 'main' into Controller/DirectMessage

JackyKredo commented on resources/views/users/items/add.blade.php:
```diff
@@ -120,4 +122,28 @@
     </form>
     @include('users.categories.modals.create_category')
 </div>
+
```
Separate scripts from blade file to js file

Mavo39 replied:
I have created new js files inside public holder. Please check it.

Mavo39 committed:
[Controller/DirectMessage] Add js files
Changes:

Add section on add and edit item page to import script
Create new js files to use jQuery
